.github/workflows/ci_ubuntu.yaml

```diff
@@ -14,12 +14,12 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        libt_version: ["2.0.5", "1.2.15"]
+        libt_version: ["2.0.6", "1.2.16"]
         qbt_gui: ["GUI=ON", "GUI=OFF"]
-        qt_version: ["5.15.2", "6.2.0"]
+        qt_version: ["5.15.2", "6.3.0"]
```

Comment:
I think here we should still test build the minimum supported qt6 version which is 6.2.0.
The windows/macos CI can test build the latest Qt6 version no problem.

Reply:
No problem, changed it back to `6.2.0`

Reply:
BTW, can you or @glassez upload `updated libraries` at some stage to `AppVeyor`?

Reply:
>BTW, can you or @glassez upload updated libraries at some stage to AppVeyor?

Testing against older (yet supported) library versions could be valuable too...